fastlane/Fastfile

```diff
@@ -86,6 +86,34 @@ platform :ios do
     Spaceship::Tunes::SandboxTester.create!(email: email, password: password)
   end
 
+  desc "Deployment checks"
+  lane :deployment_checks do |options|
+    check_pods
+    carthage_archive
+    check_no_github_release_exists
```

Comment:
Couldn't think of other stuff to check that wouldn't be covered by integration tests, but I think this would be pretty helpful to have